cf/irsa-api-role.yaml

```diff
@@ -212,3 +212,17 @@ Resources:
                   - "cognito-idp:DescribeUserPool"
                   - "cognito-idp:DescribeUserPoolClient"
                 Resource: '*'
+        - PolicyName: "can-access-rds"
```

Comment:
I noticed some templates have spaces between `PolicyName` and some don't. What do you prefer? just asking to see what's the general feeling and change it in the future

Reply:
Personally I prefer the spaces

Reply:
I'll update the api-role with the spaces